schemas/questionset/1.0/schema.json

```diff
@@ -755,7 +755,33 @@
         "Standalone Assessment",
         "Moderated Assessment"
       ]
+    },
```

Comment:
this is wrong. I suggest you to use a JSON editor... new attributes should be added inside properties object. and additionalProperties field should be outside of it...